bin/install

```diff
@@ -134,13 +134,12 @@ get_java() {
 			-o $destdir/${variant}.sig $signature
 
 		# gpg servers are known not to be reliable
-		gpg --keyserver hkp://ha.pool.sks-keyservers.net:80  --recv $sigkey \
-		|| gpg --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv $sigkey \
-		|| gpg --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv $sigkey \
-		|| gpg --keyserver hkp://ipv4.pool.sks-keyservers.net:80 --recv $sigkey \
-		|| gpg --keyserver hkp://ipv4.pool.sks-keyservers.net:80 --recv $sigkey \
-		|| gpg --keyserver hkp://ipv4.pool.sks-keyservers.net:80 --recv $sigkey
-		if gpg --verify $destdir/${variant}.sig ; then
```

Comment:
> 		if gpg --verify $destdir/${variant}.sig ; then
> 			echo "Archive verification: successful"

@BaCaRoZzo We removed the `if` but left the echo, so we are not doing any Archive verification? Just mentioning to make sure it was intentional. :-)

Reply:
Oh man!!
I tested that by changing the plugin locally and then copying the code to the checked out repo...sorry about that. :man_facepalming: 

Going to fix it.